substrate/frame/staking/src/benchmarking.rs

```diff
@@ -50,7 +52,7 @@ type MaxNominators<T> = <<T as Config>::BenchmarkingConfig as BenchmarkingConfig
 // read and write operations.
 pub fn add_slashing_spans<T: Config>(who: &T::AccountId, spans: u32) {
 	if spans == 0 {
-		return
+		return;
```

Comment:
Please undo formatting and use the exact same formatting version as in the CI: `cargo +nightly-2024-09-11 fmt`

Reply:
I'll fix it; next time, how do I check which version to use?

The [`quick-checks / fmt (pull_request)`](https://github.com/paritytech/polkadot-sdk/actions/runs/11315135576/job/31465979882?pr=6025) job only shows `cargo +nightly fmt --all -- --check`.

Reply:
It is set in this [env file](https://github.com/paritytech/polkadot-sdk/blob/6c3219ebe9231a0305f53c7b33cb558d46058062/.github/env).